lua/snippets/utils/builtin.lua

```diff
@@ -216,8 +216,10 @@ local function word_under_cursor(cur, line)
 end
 
 local function get_selected_text()
+	vim.print(vim.fn.visualmode())
```

Comment:
Is this actually needed? I don't see it in my normal configuration. But in my testing configuration, I'm using a debug utility from Folke's dotfiles and bind that to `vim.print` for extra information and I'm seeing `Debug: ~/projects/plugins/nvim-snippets/lua/snippets/utils/builtin.lua:218 "v"`. If it's needed indeed, then I apologize for any inconvenience and please do ignore me.

Reply:
No, you're absolutely right. I was debugging what that function returned as a sanity check for that variable and I forgot to remove it. It should be removed now. Thanks for pointing it out